fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -5195,5 +5197,11 @@ public LogicalPlan visitUseDatabase(UseDatabaseContext ctx) {
         return ctx.catalog != null ? new UseCommand(ctx.catalog.getText(), ctx.database.getText())
                 : new UseCommand(ctx.database.getText());
     }
+
+    @Override
+    public LogicalPlan visitShowConvertLsc(ShowConvertLscContext ctx) {
+        MultipartIdentifierContext database = ctx.database;
+        return database == null ? new ShowConvertLSCCommand(null) : new ShowConvertLSCCommand(database.getText());
```

Comment:
use visitMultipartIdentifier to parse database, in case of user write sql like: catalog.database